ckanext/scheming/validation.py

```diff
@@ -83,6 +88,27 @@ def validator(key, data, errors, context):
     return validator
 
 
+@scheming_validator
+def scheming_isodatetime(value, context):
+    def validator(key, data, errors, context):
```

Comment:
You could write this as a simple one or two parameter validator instead. Those validators can return the value they've converted. This validator won't do anything because four-parameter validators need to set their value back in data[key] to see any change. http://docs.ckan.org/en/latest/extensions/adding-custom-fields.html?highlight=custom%20validator#custom-validators